performance/cinterop/src/main/kotlin-native/org/jetbrains/cinteropBenchmarks/macrosBenchmark.kt

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2010-2019 JetBrains s.r.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jetbrains.macrosBenchmarks
+import kotlinx.cinterop.*
+
+const val benachmarkSize = 1000
+
+actual fun macrosBenchmark() {
```

Comment:
Btw, this benchmark doesn't seem to bench macros, C macros don't have any runtime cost.
So what is this benchmark for actually?

Reply:
Used macros produced function and structs. Macros itself has no runtime cost. Do you mean that I should remove it or rename?

Reply:
Rename or include into other benchmark.